### ISSUE TITLE
main: Home page could be better

### ISSUE THREAD

**Issue by baktrius** [baktrius/nhexonline#11]
Home page currently shows some welcome message which isn't very helpful.
It could contains shortcuts to other parts of the page.

**Comment by Liir6676** [comment]
Proponuję zainspirować się https://shobu.io/
1. Lista graczy online
2. Czat byłby miły, ale przy założeniu ograniczeń zasobów czasowych nie najistotniejszy (niski priorytet)
3. Dymek z changlogiem prawdopodobnie nie jest potrzebny dla nhex
4. Link do githuba do sekcji zgłaszania błędów- popularyzowanie zgłaszania błędów na githubie (wysoki priorytet)
5. Tak jak w pierwszej wersji serwisu i jak na shobu.io proponuję zaimplementować tworzenie stołów i ich listę na stronie głównej (najwyższy priorytet)(zachęcam do zapoznania się z tym jak działa Tworzenie stołu na shobu.io, może to być inspirujące)

**Comment by baktrius** [comment]
Adresując 1,2 nie mam dużej motywacji do rozwiania serwisu w stronę zrzeszania i organizowania fanów gry. Uważam, że istnieją odpowiednie platformy i miejsca do tego typu aktywności jak komunikatory internetowe czy media społecznościowe. Zachowanie prywatności jak ktoś nie chce, żeby go było widać na jakiejś liście według mnie na plus. Ostatecznie publiczny user created content (w tym nicki graczy) jest problematyczny i wymaga moderowania.

3,4 wydaje mi się, że można rozwiązać ogólniej dodając sekcję newsów na głównej stronie do zarządzania przez administratora serwisu z poziomu panelu administracyjnego. To czy dodać opis ostatnich zmian, czy instrukcję zgłaszania błędów to może być wtedy indywidualna decyzja każdego admina. Czy to brzmi dobrze?

5 sekcja z szybką możliwością stworzenia stołu i grania jest rozsądnym pomysłem, jako że to podstawowa funkcjonalność strony. Do dodania.

**Comment by Liir6676** [comment]
Ad. 1 i 2 faktycznie przerost formy nad treścią. Również jestem za minimalistycznym podejściem. Nie znalazłem porządnych argumentów za implementacją listy i chatu.
Ad. 3, 4, 5 po przemyśleniu proponuję dodać np. modyfikowalny prosty footer, w którym znajdą się linki do repo (github) i changelog+dyskusja+propozycje stojące za linkiem do kanału discorda (czy to w postaci PNG, czy choćby ahref- przy najechaniu na takowe podświetlałoby odpowiednio "zgłoś błąd" i np. "rozwój serwisu". Wówczas możnaby się skupić w większej mierze na tym, by ekran startowy posiadał możliwość tworzenia stołów.

**Comment by baktrius** [comment]
Na czym polega "modyfikowalność" footera. Tzn. jak wyobrażasz sobie jego konfigurowanie? Czy ma to robić administrator strony czy ma być zapisane na stałe w projekcie.
Jedna opcja (gdzie administrator strony zarządza footerem) jest taka, że byłby jakaś sekcja footer links w panelu administracyjnym. I to co tam byś dodał pojawiałoby się na dole strony. Czy to brzmi dobrze?

**Comment by Liir6676** [comment]
Tak, dokładnie. To brzmi dobrze. Z poziomu administracyjnego- jeśli to możliwe- możliwośc edytowania footera html i css (aby zostawić estetykę footera na barkach administratora). Jeśli pola html i css będą puste, to czy footer nie będzie istniał? Jeśli zawsze będą jego pozostałości to proponuję dodać flagę (footer ON/OFF). Może to przerost formy nad treścią. Może jest inny sposób, by każdy administrator w jakimś stopniu dostosowywał wygląd strony?

**Comment by baktrius** [comment]
Jak chcesz mieć pełną swobodę w edycji footera na poziomie html i css to chyba najprościej abyś zmienił sobie kod projektu. Myślałem, żeby administrator mógł z admina dostosowywać linki wyświetlane na stronie. Spróbuję coś zrobić i zobaczymy czy będzie ok.

**Comment by baktrius** [comment]
Some progress regarding quick play section on main page has been done in 05324a4.

Można jeszcze ulepszyć tę sekcję. Myślałem aby stworzenie nowego stołu w tej sekcji nie przekierowywało od razu do szczegółów stołu tylko aktualizowało rubrykę na głównej stronie z podstawowymi informacjami dołączenia do stołu.
Dodatkowo rubryka ta zawierałaby link do strony z pełnymi szczegółami stołu.

**Comment by baktrius** [comment]
Added footer with admin configurable links in bf8f25e.
